WalletWasabi.Tests/LiveServerTests.cs

```diff
@@ -53,26 +62,91 @@ public class LiveServerTests : IClassFixture<SharedFixture>
 		[InlineData(NetworkType.Testnet)]
 		public async Task GetAllRoundStatesAsync(NetworkType networkType)
 		{
-			using (var client = new SatoshiClient(_networkUriMappings[networkType]))
+			using (var client = new SatoshiClient(LiveServerTestsFixture.UriMappings[networkType]))
 			{
 				var states = await client.GetAllRoundStatesAsync();
 				Assert.True(states.NotNullAndNotEmpty());
 				Assert.True(states.Count() >= 1);
 			}
 		}
 
-		// Offchain
+		#endregion Blockchain
+
+		#region ChaumianCoinJoin
+
+		[Theory]
+		[InlineData(NetworkType.Testnet)]
+		public async Task RegisterAliceInputThenUnConfirmAsync(NetworkType networkType)
+		{
+			(BitcoinPubKeyAddress activeOutputAddress, BitcoinPubKeyAddress changeOutputAddress, string blindedDataHex, string proof, List<TxoRef> utxos) = LiveServerTestsFixture.GetAliceInputData(networkType);
+
+			// blinded activeOutputAddress.ScriptPubKey
+			byte[] blinded = ByteHelpers.FromHex(blindedDataHex);
+
+			var inputProofModels = utxos.Select(txrf => new InputProofModel
+			{
+				Input = txrf,
+				Proof = proof // blinded.BlindedData signed with private key owning utxos
+			});
+
+			var aliceClient = await AliceClient.CreateNewAsync(changeOutputAddress, blinded, inputProofModels, LiveServerTestsFixture.UriMappings[networkType]);
```

Comment:
Isn't it IDisposable? Can you put it into a using block?

Reply:
Yep, updated.